lychee-bin/src/formatters/response/mod.rs

```diff
@@ -25,4 +25,12 @@ pub(crate) const MAX_RESPONSE_OUTPUT_WIDTH: usize = 10;
 pub(crate) trait ResponseFormatter: Send + Sync {
     /// Format the response body into a human-readable string
     fn format_response(&self, body: &ResponseBody) -> String;
+
+    /// Detailed response formatter (defaults to the normal formatter)
+    ///
+    /// This is used for some output modes to provide more detailed information
+    /// and in verbose mode.
```

Comment:
I find the sentence a bit confusing, especially `and in verbose mode`.

Reply:
Rephrased. Better now?

Reply:
yes :+1: